net-misc/sysrepo/sysrepo-1.4.104.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib
+
+DESCRIPTION="YANG-based configuration and operational state data store"
+HOMEPAGE="https://www.sysrepo.org"
+SRC_URI="https://github.com/sysrepo/sysrepo/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+# tests trigger sandbox violation
+RESTRICT="test"
+
+RDEPEND="net-libs/libyang:=[${MULTILIB_USEDEP}]"
```

Comment:
why do you actually need `MULTILIB_USEDEP` for new packages?

Reply:
I figured it could be useful since it's already added in libyang. But I don't need it, so AFAIC it can be dropped